### DISCUSSION TITLE
use NDI Driver as NDI 5 HX Input

### DISCUSSION THREAD

**Discussion by cyclemat** [DistroAV/DistroAV#973] [Ideas]
my idea was to use the ndi driver from the NDI tools to realize an HX input in OBS is it possible

**Comment by Trouffman** [comment]
NDI HX requires aa specific SDK (Adavanced) and would require everyone to get their own licence as well.

Reading NDI HX is one thing, encoding and sending NDI HX is another challenge/

**Reply by cyclemat** [reply]
But when WE have an hx Input WE can use the hx screencapture Software to send to obs

**Comment by Trouffman** [comment]
the suggested approach is not valid.
NDI HX driver : "Enables playback of HX sources"

We can already do that, ( as in "read/decode the HX stream").

The request is to sent HX, this is not achieved by the driver, but the Advanced SDK.
